cmd/oras/pull.go

```diff
@@ -185,3 +200,7 @@ func runPull(opts pullOptions) error {
 func isManifestMediaType(mediaType string) bool {
 	return mediaType == docker.MediaTypeManifest || mediaType == ocispec.MediaTypeImageManifest
 }
+
+func generateKey(desc ocispec.Descriptor) string {
```

Comment:
Can we have a more meaningful name other than `generateKey`? Meanwhile, can we document this method so that others can understand easily?

Reply:
How about `namedDigest`?

Reply:
Changed to name` generateDeduplicatedMapKey` and added a doc comment.

Reply:
Changed to name `generateContentKey` and added a doc comment.